decidim-core/spec/lib/assets/tailwind_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "decidim/assets/tailwind"
+
+module Decidim
+  module Assets
+    describe Tailwind do
+      let(:app_path) { Rails.application.root.to_s }
+
+      describe ".write_runtime_configuration" do
+        let(:runtime_config_path) do
+          Rails.application.root.join("tailwind.config.js")
+        end
+
+        it "generates the runtime configuration" do
+          create_instance
+
+          expect(File.exist?(runtime_config_path)).to be(true)
+        end
+
+        it "adds decidim gems" do
+          create_instance
+
+          configuration = File.read(runtime_config_path)
+          expect(configuration).to include("decidim-core")
```

Comment:
I'd test the actual path for the core gem here in order to ensure it catches the bug there is currently with it.